Makefile

```diff
@@ -34,7 +36,7 @@ ARCH = $(shell go env GOARCH)
 all: manager readinessServer
 
 export_env_vars:
-export NAMESPACE = openshift-storage
+export NAMESPACE ?= openshift-storage
```

Comment:
This is a downstream namespace, I don't think we want to include it in an upstream project. 
I guess the default should be `default`